Makefile

```diff
@@ -0,0 +1,13 @@
+include env.make
+
+create-pip-env:
+	pipenv install
+
+setup:
+	sudo -H pip install -U pipenv
```

Comment:
`sudo -H pip3 install -U pipenv`
Or is there a better way to use pip3? Maybe setting it to default in the env.make?